src/pages/Datasets.jsx

```diff
@@ -7,30 +7,40 @@ import { DatasetsTableRow } from '../components/Datasets/DatasetsTableRow';
 import { useClient } from '../context/client-context';
 import { useSnackbar } from 'notistack';
 import { getSnackbarOptions } from '../components/Common/utils/snackbarOptions';
+import { compareSemver } from '../lib/common-helpers';
 
 function Datasets() {
   const [datasets, setDatasets] = useState([]);
   const { client: qdrantClient } = useClient();
   const [isLoading, setIsLoading] = useState(false);
   const { enqueueSnackbar, closeSnackbar } = useSnackbar();
   const errorSnackbarOptions = getSnackbarOptions('error', closeSnackbar);
+  const { client } = useClient();
 
   useEffect(() => {
     const fetchData = async () => {
       setIsLoading(true);
       try {
+        const telemetry = await client.api('service').telemetry();
```

Comment:
we already receive qdrant's version on the top level to display it in the left bottom corner.
Would it be possible to reuse it?

Reply:
Fixed